ross/materials.py

```diff
@@ -115,6 +197,23 @@ def load_data():
 
     @staticmethod
     def use_material(name):
+        """Function to load the materials properties and instantiate a Material Object.
+
+        Parameters
+        ----------
+        name : Material's name.
```

Comment:
In order to keep up with the standard we're using, this should be:

name: string
        Material's name.